target/linux/layerscape/armv8_64b/config-default

```diff
@@ -0,0 +1,4667 @@
+#
```

Comment:
Please run "make kernel_oldconfig CONFIG_TARGET=subtarget" when this subtarget is selected to reduce the size of this configuration file.